examples/nemo/scripts/utils.py

```diff
@@ -151,8 +146,9 @@ def enhance_tree_information(schedule):
             _it_should_be(reference.symbol, ScalarType, REAL_TYPE)
         elif isinstance(reference.symbol.interface, ImportInterface) and \
                 reference.symbol.interface.container_symbol.name == "phycst":
-            # Everything imported from phycst is a REAL
-            _it_should_be(reference.symbol, ScalarType, REAL_TYPE)
+            if not isinstance(reference.symbol, RoutineSymbol):
+                # Every datasymbol imported from phycst is a REAL
+                _it_should_be(reference.symbol, ScalarType, REAL_TYPE)
```

Comment:
Could you add to this comment to explain why chasing the imports from phycst hasn't fixed this?

Reply:
If fact it did! I submitted a CI with most of these deleted and the performance and results are still the same :)

I left the are_integers because some of them come from transitive imports, and I explained it in the docstring.